docs/u-boot.md

```diff
@@ -0,0 +1,32 @@
+# U-Boot Support
+MilvusVisor has support for U-Boot environment.
+We tested this on QEMU.
+
+## How to build
+To enable U-Boot support, you need to enable `u-boot` feature flag.
```

Comment:
"you need to add `u-boot` feature flag on building binaries of hypervisor and bootloader"